server/fileupdater.mjs

```diff
@@ -318,4 +319,11 @@ function v6cssPieces(properties) {
       return;
     }
   }
+
 }
+
+function v7CanvasLayer(properties) {
+  if (properties.layer < -3){
+    properties.layer-=1
+  }
```

Comment:
Apparently I didn't submit my comment.

This is not enough because it doesn't undo the new default layer for canvas.

IMO, the file updater should try to change as little as possible. I would only update the layer of canvas (to the old default if it is null; delete it if it was -4). That would make the layer changes you do unnecessary.